proofpoint: improve deprecated notice (#11159)
demisto/content: Packs/ProofpointServerProtection/Integrations/ProofpointServerProtection/ProofpointServerProtection_description.md

Users must be assigned to the admin role to use this integration.

> **Note:** This is a beta Integration that has been deprecated. The Integration uses a scraping API which is not supported anymore. We do not recommend using this integration in production environments. Some commands may not work as expected. Use the **Proofpoint Protection Server v2** Integration instead.

Reviewer: ```suggestion
**Note:** This is a beta Integration that has been deprecated. The Integration uses an unsupported scraping API. We do not recommend using this integration in production environments. Some commands may not work as expected. Use the **Proofpoint Protection Server v2** Integration instead.
```
